packages/react-components/src/theming/icons.tsx

```diff
@@ -74,6 +78,10 @@ export const DEFAULT_COMPONENT_ICONS = {
   ControlButtonParticipants: <People20Filled />,
   ControlButtonScreenShareStart: <ShareScreenStart20Filled />,
   ControlButtonScreenShareStop: <ShareScreenStop20Filled />,
+  /* @conditional-compile-remove(file-sharing) */
+  Cancel: <Dismiss16Regular />,
+  /* @conditional-compile-remove(file-sharing) */
+  Download: <ArrowDown16Regular />,
```

Comment:
I think this should be `<ArrowDownload16Regular>`: 
https://iconcloud.design/browse/Fluent%20System%20Library/Fluent%20Regular/60e878c1e-3821b375b

Instead of `ArrowDown16Regular`:
https://iconcloud.design/browse/Fluent%20System%20Library/Fluent%20Regular/80d07ae9f-3821b375b

Reply:
Oh what would I do without you @JamesBurnside :D